packages/cf-style-container/test/index.js

```diff
@@ -11,22 +11,33 @@ import { variables } from 'cf-style-const';
 import renderer from 'react-test-renderer';
 import felaTestContext from '../../../felaTestContext';
 
+test('mergeThemes should return an immutable and deeply cloned object', () => {
+  const themeA = () => ({ color: 'yellow' });
+  const themeB = { color: 'blue', breakpoints: { desktop: '1em' } };
+  const props = mergeThemes(variables, themeA, themeB);
+  expect(() => {
+    props.theme.color = 'white';
+  }).toThrow();
+  expect(props.theme.color).toEqual('blue');
+  expect(props.theme.breakpoints).not.toBe(themeB.breakpoints);
```

Comment:
Ah, reading the test seems to indicate that this shouldn't be the same when in fact is, right? Could we make it more clear that we are testing for "reference equality"? Maybe add a comment or something?